signature-aggregator/aggregator/aggregator.go

```diff
@@ -151,7 +151,13 @@ func (s *SignatureAggregator) CreateSignedMessage(
 				)
 			}
 		}
+		s.metrics.SignatureCacheHits.Add(float64(len(signatureMap)))
```

Comment:
What's the reasoning behind incrementing based on the number of signatures (i.e. the cache value) rather than the number of key hits? Signature-based metrics would make it more difficult to identify appropriate cache sizes based on usage.

Reply:
Not my PR but not sure what do you mean by this. The `signatureMap` here is just the number of key hits from my understanding.

Reply:
The cache is keyed by `warpMessageID`, but we're adding metrics based on the size of the corresponding value. If the Warp message is signed by 1 or 1000 validators, it's still a cache hit or miss, but the measuring the number of signatures would tell a wildly different story than measuring the actual cache hit/miss rate.

Reply:
as discussed on standup, I left the metric in place on a per-signature basis. also, I changed the "cache miss" metric to reflect the number of signatures that need to be requested from the network, in https://github.com/ava-labs/awm-relayer/pull/415/commits/70b7722ea01cdd358da4568073638bdcfa67e610